force-app/main/default/classes/UTIL_OrgTelemetry.cls

```diff
@@ -466,5 +467,12 @@ public without sharing class UTIL_OrgTelemetry {
                     AND CreatedDate = LAST_N_DAYS:365
             ]
         );
+
+        featureManager.setPackageIntegerValue(
+            TelemetryParameterName.Data_CountAutoEnrollmentMappings.name(),
+            [
+                SELECT COUNT() FROM Affl_Mappings__c WHERE Auto_Program_Enrollment__c = true
+            ]
+        );
```

Comment:
In an ideal world, I would want this isolated into its own method with a name that states its purpose. The state of the old code does not tell an effective story. about what we are doing nor separate the concerns.

This is also a location where we might be well served by having the SOQL query assign to a local variable before setting the value, again for readability and purpose.

Note that this is more stylistic. Note that keeping it like this can help as multiple assignments to the same feature parameter during a test suite run can cause row lock issues.

Reply:
Readability enhanced by introducing metrics model wrapper ([reference commit](https://github.com/SalesforceFoundation/EDA/pull/1928/commits/0e5228f8955451722bc44f18be7032c81d3b61ea)).